ct3/Menu/src/menu/QLGT.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package menu;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ *
+ * @author Admin
+ */
+public class QLGT implements ChucNang {
+    private List<PTGT> list;
+    private Scanner sc = new Scanner(System.in);
+
+    public QLGT() {
+        list = new ArrayList<>();
+    }
+    private int tonTai (String ma) {
+        for(int i = 0; i < list.size(); i++) {
+            if (list.get(i).getMa().equalsIgnoreCase(ma))
+                return i;
```

Comment:
Nếu câu lệnh tiếp theo của if (tương tự for, while,...) nằm trên cùng 1 dòng thì mới được phép bỏ ngoặc nhọn, nếu không nhất định phải dugf ngoặc nhọn bao lấy câu lệnh.